frontend/package.json

```diff
@@ -8,7 +8,7 @@
     "build": "tsc && vite build",
     "lint": "eslint src --ext ts,tsx --max-warnings 0",
     "preview": "vite preview",
-    "generate-client": "openapi --input http://localhost:8000/openapi.yml --output ./src/api --client axios"
+    "generate-client": "openapi --input http://localhost:8000/openapi.yml --output ./src/api --exportServices false"
```

Comment:
We don't use any of the generated service definitions so i figured we can just remove them from the repo for now

Reply:
Makes sense to me. Good to remove bloat.